packages/backend/server/src/plugins/indexer/config.ts

```diff
@@ -31,6 +31,7 @@ defineModuleConfig('indexer', {
   enabled: {
     desc: 'Enable indexer plugin',
     default: true,
+    env: ['AFFINE_INDEXER_ENABLED', 'boolean'],
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/hTwOityLamd4hitrae7M/99c80b41-f836-4344-8bbf-5f13ca4d4e54.png)

Default will be `false`?